<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OsteotomyPlanner</class>
 <widget class="qMRMLWidget" name="OsteotomyPlanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>586</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="qMRMLSubjectHierarchyComboBox" name="SubjectHierarchyComboBox">
     <property name="defaultText">
      <string>Select model folder</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="qMRMLSubjectHierarchyTreeView" name="SubjectHierarchyTreeView">
     <attribute name="headerCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox">
     <property name="title">
      <string>Actions</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="MoveButton">
        <property name="text">
         <string>Move</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SplitButton">
        <property name="text">
         <string>Split</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="ActionsWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="MoveActionWidget" native="true">
      <attribute name="title">
       <string>Move</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
     <widget class="QWidget" name="SplitActionWidget" native="true">
      <attribute name="title">
       <string>Split</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="AutoPlaneButton">
         <property name="toolTip">
          <string>Place the splitting plane at the center of mass of the model</string>
         </property>
         <property name="text">
          <string>Auto-place split plane</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ManualPlaneButton">
         <property name="toolTip">
          <string>Begin placing splitting plane as a fiducial</string>
         </property>
         <property name="text">
          <string>Begin manual placement of plane</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ClearPlaneButton">
         <property name="toolTip">
          <string>Clear current splitting plane</string>
         </property>
         <property name="text">
          <string>Clear plane</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="SplitFinishWidget">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="SplitConfirmButton">
            <property name="toolTip">
             <string>Confirm split</string>
            </property>
            <property name="text">
             <string>Confirm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SplitCancelButton">
            <property name="toolTip">
             <string>Cancel split</string>
            </property>
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkComboBox</class>
   <extends>QComboBox</extends>
   <header>ctkComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSubjectHierarchyTreeView</class>
   <extends>QTreeView</extends>
   <header>qMRMLSubjectHierarchyTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLSubjectHierarchyComboBox</class>
   <extends>ctkComboBox</extends>
   <header>qMRMLSubjectHierarchyComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>OsteotomyPlanner</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>SubjectHierarchyComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
